pages/Add/index.js

```diff
@@ -1,7 +1,131 @@
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import {
+  setLocalStorage,
+  loadLocalStorage,
+} from "../../components/LocalStorage";
+import { nanoid } from "nanoid";
+
 export default function Add() {
+  const [startDate, setStartDate] = useState(new Date());
+  const [fishList, setFishList] = useState(
+    loadLocalStorage("localfFishList") ?? []
+  );
+
+  useEffect(() => {
+    setLocalStorage("localfFishList", fishList);
+  });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const newFish = {
+      id: nanoid(),
+      fishname: e.target.name.value,
+      fishweight: e.target.weight.value,
+      fishlength: e.target.length.value,
+      location: e.target.location.value,
+      date: startDate,
+    };
+    setFishList([...fishList].concat(newFish));
```

Comment:
Didn't know you could do it with 'concat' - I would have done it like this: 
```suggestion
    setFishList([...fishList, newFish]);
```

Reply:
Yeah i like the concat on this one because it is more intuitive to me. I can remember the syntax far easier.
But of course does your solution work aswell :)